src/components/analyzer/component.jsx

```diff
@@ -87,27 +90,25 @@ class Analyzer extends PureComponent {
       analysis: { data, loading, downloadUrl }
     } = this.props;
     const { fileLoading } = this.state;
-    const btnClass = classnames(
-      'c-btn -light apply-analysis-btn',
-      { '-disabled': !points.length }
-    );
 
     return (
-      <div className="l-analyzer">
-        <div className="c-analyzer">
-          {(!!data.length && !loading) && (
-            <div className="analyzer-header">
-              <button
-                type="button"
-                onClick={() => { this.triggerExpandedTableModal(); }}
-              >
-                <Icon
-                  name="icon-expand-window"
-                  className="expand-table-icon"
-                />
-              </button>
-            </div>
-          )}
+      <Layout
+        disableApply={!points.length}
+        onApply={() => this.onApplyAnalysis()}
+      >
+        {(!!data.length && !loading) ? (
+          <DownloadableTable
+            onExpandTable={() => this.triggerExpandedTableModal()}
+            downloading={fileLoading}
+            downloadDisabled={!(downloadUrl && !loading)}
```

Comment:
Might be clearer to say 
```suggestion
            downloadDisabled={!downloadUrl || loading}
```